src/services/game/showGamePanel.js

```diff
@@ -1,4 +1,18 @@
-export const showGamePanel = () => {
-  const startGamePanel = document.querySelector('.main');
-  startGamePanel.innerHTML = '';
+import { createQuizGameView } from '../../components/quizGameView/quizGameView';
+
+export const showGamePanel = async () => {
+  const mainView = document.querySelector('.main-view');
+  const startGamePanel = document.querySelector('.main-game-panel');
+  const spinner = document.querySelector('.sk-circle');
+
+  startGamePanel.remove();
+  spinner.classList.remove('hide');
+
+  const ONE_SECOND_MILLIS = 1000;
```

Comment:
why upper snake case? it might be used while variables are globally available, but there they are assigned to one function due to declaration
you can declare those variables in separate file, or above the function scope in case that we would use them in other place

but we can avoid to use `SPINNER_VISIBILITY_TIME ` and create helper function in case that we want to add different spinner time `getSpinnerVisibilityTime = (onesecondmillis, 5) => onesecondmillis * 5`